wiki/Skinning/de.md

```diff
@@ -1,28 +1,28 @@
 # Skinning
 
-Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen oder ihre eigenen Skins zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, fokussiert auf "pro-Spielbarkeit", Minimalismus, oder nahezu unspielbar.
+Skinning ist eine der Hauptfunktionen von osu! und ermöglicht Spielern den Standard-Skin ihren eigenen Wünschen entsprechend anzupassen oder ihre eigenen Skins zu erstellen! Deshalb gibt es Skins in allen Formen und Farben: sei es spaßeshalber, mit Fokus auf professioneller Spielbarkeit, Minimalismus, oder nahezu unspielbar.
 
 ## Skinsets
 
 *Hauptseite: [Skinsetliste](/wiki/Ranking_criteria/Skin_set_list)*
 
-**Nur für Beatmaps**: Wenn dein Beatmap-Skin ein einziges Element aus den Sets enthält, die in den Ranking-Kriterien aufgelistet werden, muss er auch alle anderen Elemente aus dem besagten Skin-Set enthalten. Das trifft nur auf Beatmap-Skins zu, aber andere Skinersteller können diese Listen auch in Betracht ziehen.
+**Nur für Beatmaps**: Wenn dein Beatmap-Skin ein einziges Element aus den Sets enthält, die in den Ranking-Kriterien aufgelistet werden, muss er auch alle anderen Elemente aus dem besagten Skin-Set enthalten. Das trifft nur auf Beatmap-Skins zu, aber andere Skinersteller können sich potentiell an diesen Listen orientieren.
 
 ## Liste der Skinelemente
 
 Die Liste der Skinelemente ist in diese Kategorien aufgeteilt:
 
-- [Interface-Skinning](/wiki/Skinning/Interface)
+- [Audio-Skinning](/wiki/Skinning/Sounds)
```

Comment:
Why did you move this to the top?

Reply:
A lot of lists of this kind in English articles are sorted alphabetically. I think it makes sense to incorporate that in the German wiki as well (French does it too). See for example the [recent reordering](https://github.com/ppy/osu-wiki/pull/12866/commits/469e94caf55d3ed590e1d28a90bcab371bbfc3fd) in `Help centre`.

Now you could say, `Skinning der Benutzeroberfläche` needs to be at the bottom of the list. I put it here because it looks weird having the game mode skinning articles "as a group" in between. Are you okay with this change?

Reply:
I see. Yeah I'm okay with this